migrations/postgres/20250609154246_dataset_entries_remove_cross-domain_cascade.sql

```diff
@@ -0,0 +1,14 @@
+/* ------------------------------ */
+
+-- NOTE: There should be no cross-domain strong refs (and therefore cascades).
+
+-- "dataset_entries_owner_id_fkey" is obtained by a query:
+-- SELECT conname
+-- FROM pg_constraint
+-- WHERE conrelid = 'dataset_entries'::regclass AND contype = 'f';
```

Comment:
It this really important comment?

Reply:
I think so, because there is a constraint name that is not obvious where it comes from